### PR TITLE
Run PHP Tests on GitHub Actions

### PR DESCRIPTION
**Pull request by lucatume**
Update the repository to run tests on `slic` and use GitHub Actions to run PHP tests in CI.

### CODE REVIEW

**Comment by lucatume** [comment]
Passing tests [here](https://github.com/the-events-calendar/tribe-common/actions/runs/3008058312)

File: tests/wpunit/Tribe/DependencyTest.php [hunk 1]
```diff
@@ -49,7 +49,7 @@ public function validClassToPueProvider() {
 
 	/**
 	 * @test
-	 * @dataProvider validClassToPueProvider
+	 * @dataProvider valid_calls_to_pue_provider
```
Comment (Camwyn) [inline tests/wpunit/Tribe/DependencyTest.php]: Sanity check - was the witch from `Class` to `calls` intentional?

File: tests/wpunit/Tribe/DependencyTest.php [hunk 2]
```diff
@@ -162,22 +162,22 @@ public function dependency_matrix() {
...
+//				'two'   => array_merge( $two, [
+//					'should_initialize' => true,
+//					'failure_message'   => 'Plugin Two should activate: its version satisfies One\'s requirements.',
+//				] ),
+//				'three' => array_merge( $three, [
+//					'should_initialize' => true,
+//					'failure_message'   => 'Plugin Three should activate: its version satisfies One\'s requirements.',
+//				] ),
+//			],
+//		];
```
Comment (Camwyn) [inline tests/wpunit/Tribe/DependencyTest.php:180]: remove commented out code?

**Review by Camwyn — COMMENTED** [review]
Just the commented out code question.

File: tests/wpunit/Tribe/DependencyTest.php (hunk 1 above)
Comment (lucatume) [reply tests/wpunit/Tribe/DependencyTest.php]: Typo, thanks.

File: tests/wpunit/Tribe/DependencyTest.php (hunk 2 above)
Comment (lucatume) [reply tests/wpunit/Tribe/DependencyTest.php:180]: I had it commented during debug, it should not be commented out, thanks!

**Review by Camwyn — APPROVED** [review]

**Pull request merged**